demos/data-lakehouse-iceberg-trino-spark/create-spark-ingestion-job.yaml

```diff
@@ -43,13 +43,13 @@ data:
         stackable.tech/vendor: Stackable
     spec:
       sparkImage:
-        productVersion: 3.5.1
+        productVersion: 3.5.2
       mode: cluster
       mainApplicationFile: local:///stackable/spark/jobs/spark-ingest-into-lakehouse.py
       deps:
         packages:
           - org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:1.5.0
```

Comment:
I guess we want to bump to 1.6.1 once we do the actual version bump just before 24.11 is out

Reply:
What did you mean  by 1.6.1?

Reply:
Done. I checked and it exists:

Reply:
So we now have some demos on Iceberg 1.5 and some on 1.6 ;). And I assume we did not test the change by running the big lakehouse demo? ;)

That's by the way why I'm not a fan of the next branch, because the person doing the actual 24.11 bumps still needs to grep for every tool and every version and bump them, as we can not rely on PRs into next (think of e.g. iceberg, postgres, minio or Redis versions). And test the demos after the bumps. And test the actual SDP bump ;)

All intermediate PRs (as this) in my personal opinion (sorry^^) just create noise and moves the next branch into an partially updated, untested state. Running the demos for every change into the next branch IMHO is not worth the effort, e.g. it takes over an hours to test the lakehouse demo.

Anyway, I will not veto any PRs going into next (as you can see I approved this one) or will tell anyone to stop, I just wanted to express my concerns :)
So no need to change or fixup anything, all of this will be done properly by the 24.11 bump  :sweat_smile: